internal/digest/digest_test.go

```
@@ -0,0 +1,191 @@
+	"strings"
+	"testing"
+	"time"
+)
+
+func loadEventsJSONL(t *testing.T, path string) []ChangeEvent {
+	t.Helper()
+	f, err := os.Open(path)
+	if err != nil {
+		t.Fatalf("open %s: %v", path, err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)